app/shared/api-services/poll/answer-factory.tests.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+describe('Answer Factory', function () {
+  var $httpBackend,
+      answer,
+      answerFactory,
+      scope,
+      pollId;
+
+  beforeEach(module('apiServices.answer.factory'));
+
+  beforeEach(inject(function(_$httpBackend_, _Answer_) {
+    answerFactory = _Answer_;
+    $httpBackend = _$httpBackend_;
+
+    scope = {
+      deletedAnswers:  [{ id: 1, answer_text: 'Go Spurs Go!' }],
+      model: {
+        id: 777,
+        answers: [{id: 2, notOnSodahead: true, answer_text: 'wingapo'}]
+      },
+      answers: [{id: 2, answer_text: 'feel the flo'}]
+    };
+  }));
+
+  describe('postAnswer()', function () {
+    it('makes a post request', function () {
+      answer = scope.model.answers[0];
+      pollId = scope.model.id;
+      answerFactory.postAnswer(answer, pollId);
+      $httpBackend.expectPOST('/cms/api/v1/answer/').respond(201);
```

Comment:
It may be good to tweak the expectations to check against what the payload of the `POST` is in addition to it making the request